docs/en/use_case/JD_recommendation_en.md

```diff
@@ -14,13 +14,23 @@ OneFlow framework leverage on the great computational power from GPU. Therefore
 Install OneFlow with the following commands：
 ```bash
 conda activate oneflow
-python3 -m pip install --pre oneflow -f https://staging.oneflow.info/branch/support_oneembedding_serving/cu102
+python3 -m pip install -f https://staging.oneflow.info/branch/master/cu112 --pre oneflow
 ```
 In addition, following Python packages need to be installed:
 ```bash
 pip install psutil petastorm pandas sklearn
+```
+ Pull Oneflow-serving docker image：
+```bash
+docker pull oneflowinc/oneflow-serving:nightly
```

Comment:
use `docker save` to save a tar, we can store it. To avoid nightly breaking changes

Reply:
relevant packages are saved here: m7-pce-gpu02:/home/gtest/oneflow-new-v-test/pkg. image size ~12G.

Reply:
`docker save myimage:latest | gzip > myimage_latest.tar.gz` can be smaller. I've generate a 6.8G image